src/PkgDeps.jl

```diff
@@ -146,12 +153,76 @@ function direct_dependencies(pkg_entry::PkgEntry)
 end
 
 function direct_dependencies(pkg_name::String; registries::Array{RegistryInstance}=reachable_registries())
```

Comment:
Yeah, they're slightly different since it's `keys` vs `values` so it's a bit hard to combine it. I'd move the stdlib checks into `_find_latest_pkg_entry` but then I don't know what to return from there; we don't have PkgEntry's for stdlibs, and if we return `nothing` or something like that, we force the caller to handle that anyway-- and it would be kind of strange to return `nothing` for stdlibs but error for other things it can't find.